### PR TITLE
Extension forms

### PR DESCRIPTION
**Pull request by CharlGottschalk**
* Added new `init()` method to extensions that gets called after the editor instance is passed to the extension.
* Added new trigger action inside `bindButtons()` that checks whether an extension has a form to open.
* Added new `showForm()` method that displays the extension's form.
* Added new `insertHTML()` method that only requires an HTML argument for use in extensions.
* Changed CSS for forms from `.medium-editor-toolbar-form-anchor` to more generic `.medium-editor-toolbar-form`.
* Changed CSS for form inputs from `.medium-editor-toolbar-anchor-input` to more generic `.medium-editor-toolbar-input`.
* Changed CSS for form save button from `.medium-editor-toolbar-anchor-save` to more generic `.medium-editor-toolbar-save`.
* Changed CSS for form close button from `.medium-editor-toolbar-anchor-close` to more generic `.medium-editor-toolbar-close`.
* Added `fixToolbar` property for fixing the toolbar to the top of the page.
* Added `.fixed-toolbar` css for `fixToolbar` property.
* Updated `setToolbarPosition()` method to center the toolbar horizontally when `fixToolbar` property is set.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,7 @@ If you want to support IE9, you will need to use a classList pollyfill, like Eli
 * __onHideToolbar__: optional callback that will be called each time the toolbar is actually hidden for this instance of medium-editor.
 * __staticToolbar__: enable/disable the toolbar always displaying in the same location relative to the medium-editor element. Default: false
 * __stickyToolbar__: enable/disable the toolbar "sticking" to the medium-editor element when the page is being scrolled. Default: false
+* __fixToolbar__: enable/disable the toolbar being "fixed" to the top of the page. Use this instead of _stickyToolbar_. Default: false
```
Comment (Dayjo) [inline README.md]: What are the benefits/differences of this over the existing sticky / static toolbar options?

Comment (CharlGottschalk) [inline README.md]: This does not use the scroll event to position the toolbar, but instead it's fixed with CSS only. Not really much of a difference I suppose. It centers the toolbar horizontally and keeps it on top regardless of scroll position or the top of the editable element.

Comment (Dayjo) [inline README.md]: Ok :) Could you not just mod the CSS of the static toolbar?

Comment (CharlGottschalk) [inline README.md]: I actually closed this pull request and opened a new one from my branch __extended-extensions__. I improved a couple of things and removed the `fixToolbar` property. This branch's builds weren't passing.

Comment (CharlGottschalk) [inline README.md]: This pull request should not be merged. Please check out the new pull request https://github.com/daviferreira/medium-editor/pull/367

**Pull request closed without merging**